README.md

```diff
@@ -181,3 +188,10 @@ Contact
 
 Twitter: [@bundesgit](https://twitter.com/bundesgit)
 
+
+More information
```

Comment:
Maybe "References" is a more suitable title?

```suggestion
References
```

Reply:
To me, "References" seems to mean something authoritative. How about just "See also"?

Reply:
I thought about something like _press coverage_? Maybe you are right that this is not really _outside press_ (someone else) but more _inside presentation_.
So I am fine with the other suggestions as well.

Reply:
```suggestion
See also
```